cabby/data/wikidata/item.py

```diff
@@ -31,25 +31,37 @@ class Entity:
 
   `url` is the URL of the entity.
   `title` is the name of the entity.
+  'wikipedia_url' is the URL of the corresponding Wikipedia entity.
+  'wikipedia_title' is the name of the corresponding Wikipedia entity.
   `location` is a Point representing the geo-location of the entity.
```

Comment:
The order of the variables in the docstring should match their appearance in the code. Also, please provide a description of 'instance'.